assignments/lambda-classes.js

```diff
@@ -1 +1,282 @@
 // CODE here for your Lambda Classes
+
+function getRandomInt(max) {
+    return Math.floor(Math.random() * Math.floor(max));
+}
+
+// Create class Person to serve as base class
+class Person {
+    constructor(person) {
```

Comment:
You could use destructuring to not repeat person 4 times here --> constructor({name, age, location, gender})